sonar-vbnet-plugin/pom.xml

```diff
@@ -267,7 +267,7 @@
                          basedir="${sonarAnalyzer.workDirectory}/SonarAnalyzer.VisualBasic"/>
                     <copy todir="${sonarAnalyzer.workDirectory}/org/sonar/plugins/vbnet">
                       <fileset
-                        dir="${project.build.directory}/../../analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/bin/${analyzer.configuration}/net46/vbnet">
+                        dir="${project.build.directory}/../../analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/bin/${analyzer.configuration}/net46/Visual Basic">
```

Comment:
I have a weird feeling around this because of spaces in the path.

This means that building the Java part will only work on Windows.

Currently it's the case because of the orchestration. Having this spaces in the file path will only make it a pain at some point in the future in case we'll want to build the project on unix boxes (like it was possible back in the days, we had a different Maven profile to download the DLLs from repox and build the SQ/SC plugin part locally on linux)

Reply:
you're right - but they need to be escaped and it's easier not to have them